lib/player.js

```diff
@@ -6570,6 +6535,37 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
     return config;
   }
 
+  /**
+   * @return {!Object}
```

Comment:
It's a shame we can't do something like `Partial` from Typescript here....

Reply:
I know, but there is no other way... :(